en/editor/publish/publish-native.md

```diff
@@ -86,7 +86,7 @@ Android requires that all APKs be digitally signed with a certificate before the
 
 #### Orientation
 
-The orientation of the device includes four types **Portrait**, **Upside Down**, **Landscape Left**, and **Landscape Right**.
+The orientation of the device includes four types **Portrait**, **Landscape Left**, and **Landscape Right**.
```

Comment:
The orientation of the device includes four types **Portrait**, **Landscape Left**, and **Landscape Right**.
-->
We may not capitalize these words in the English language because they don't refer to a person, place, thing. It is just a term in this case. However, if these values appear inside of Creator capitalized, please keep them the way you have them.

The orientation of the device includes four types **portrait**, **landscape left**, and **landscape right**.

Reply:
These words are capitalized just because these values appear inside of Creator capitalized .Like blow:

![image](https://user-images.githubusercontent.com/27424848/84453715-26d2c400-ac8b-11ea-8cc0-4c79cc68a42b.png)

But I found “four types” should changed to "three types",I will make it right.

Reply:
Thank you for checking.